src/clib/initialize_rates.c

```diff
@@ -352,15 +352,6 @@ int initialize_rates(chemistry_data *my_chemistry, chemistry_data_storage *my_ra
                           / (densityBase1 * pow(timeBase1, 3));
 
     // These always need to be allocated since we define other variables by them.
-    my_rates->LH2_N = calloc(3, sizeof(int));
-    my_rates->LHD_N = calloc(3, sizeof(int));
-    my_rates->LCI_N = calloc(3, sizeof(int));
-    my_rates->LCII_N = calloc(3, sizeof(int));
-    my_rates->LOI_N = calloc(3, sizeof(int));
-    my_rates->LCO_N = calloc(3, sizeof(int));
-    my_rates->LOH_N = calloc(3, sizeof(int));
-    my_rates->LH2O_N = calloc(3, sizeof(int));
-    my_rates->alphap_N = calloc(2, sizeof(int));
```

Comment:
Just one question. Originally, these initializations had to be here because they were used to define the dimensions of variables defined in the fortran. Without this here I was getting seg faults. Now, these are initialized as part of their associated larger structures, which happens elsewhere. Will this no longer be a problem?